traits/tests/test_file.py

```diff
@@ -0,0 +1,51 @@
+import os.path as osp
```

Comment:
This is a bit nonstandard; changing to just `import os` (with corresponding changes below).